content/vocabularies/g-sgov-slovník-pro-agendu-a824-patenty/g-sgov-slovník-pro-agendu-a824-patenty-model.ttl

```diff
@@ -0,0 +1,11056 @@
+        ]
+    ], [ a owl:Restriction;
+      owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger;
+      owl:onClass g-sgov-slovník-pro-agendu-a824-patenty-pojem:zastupuje;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-2>
+        ]
+    ], [ a owl:Restriction;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ];
+      owl:someValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:se-nachází-ve-stavu
+    ], [ a owl:Restriction;
+      owl:allValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:se-nachází-ve-stavu;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ]
+    ], [ a owl:Restriction;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ];
+      owl:someValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-v-konkurzním-řízení
+    ], [ a owl:Restriction;
+      owl:allValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-v-konkurzním-řízení;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ]
+    ], [ a owl:Restriction;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ];
+      owl:someValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-v-exekuci
+    ], [ a owl:Restriction;
+      owl:allValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-v-exekuci;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ]
+    ], [ a owl:Restriction;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ];
+      owl:someValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-v-zástavě
+    ], [ a owl:Restriction;
+      owl:allValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-v-zástavě;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ]
+    ], [ a owl:Restriction;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ];
+      owl:someValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-podnikovým-vynálezem
+    ], [ a owl:Restriction;
+      owl:allValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:je-podnikovým-vynálezem;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-1>
+        ]
+    ], [ a owl:Restriction;
+      owl:onProperty [
+          owl:inverseOf <https://slovník.gov.cz/základní/pojem/má-vztažený-prvek-2>
+        ];
+      owl:someValuesFrom g-sgov-slovník-pro-agendu-a824-patenty-pojem:původce-vynalezl-vynález
+    ], [ a owl:Restriction;
```

Comment:
vazba nedává smysl. Původce -> původce vynalezl vynález -> Patentový spis. Chápal bych to spíše tak, že v patentovém spisu bude vazba na původce, vazba by tedy měla platit obráceně (to samé platí pro přihlašovatele a majitele)